programs/sbf/Makefile

```diff
@@ -0,0 +1,13 @@
+SBF_SDK_PATH := ../../sdk/sbf
+SRC_DIR := c/src
+OUT_DIR := target/sbf-solana-solana/release
+
+test: rust all
+	SBF_OUT_DIR=$(OUT_DIR) cargo test --features="sbf_rust,sbf_c" $(test)
```

Comment:
nit: I think this should be `$(TEST_ARGS)`

The convention is to have variables be uppercase, and the variable can contain more than just one test name. For example I would run `make test TEST_ARGS="sometest -- --nocapture"`